src/entry.ts

```diff
@@ -19,9 +19,81 @@ import {Service} from '@google-cloud/common-grpc';
 const EventId = require('eventid');
 import * as extend from 'extend';
 import * as is from 'is';
+import {StackdriverHttpRequest} from './http-request';
 
 const eventId = new EventId();
 
+export interface MonitoredResource {
```

Comment:
What are your thoughts on getting these auto-generated from protobuf? As an example see what we do in the [Stackdriver Profiler agent](https://github.com/googleapis/cloud-profiler-nodejs/tree/master/proto).

To review this, I need to open the protobuf and then verify that you actually did a good job as a protobuf to TS compiler. I would much rather have this auto-generated and correct by construction.

WDYT?

Reply:
My thought is I like that :) How are y'all generating these?

Reply:
https://github.com/googleapis/nodejs-logging/pull/314